src/gwt/src/org/rstudio/studio/client/workbench/views/console/shell/assist/RCompletionManager.java

```diff
@@ -787,7 +787,14 @@ public boolean previewKeyPress(char c)
          boolean canAutoPopup = checkCanAutoPopup(c, userPrefs_.codeCompletionCharacters().getValue() - 1);
          
          // Attempt to pop up completions immediately after a function call.
-         if (c == '(' && !isLineInComment(docDisplay_.getCurrentLine()))
+         Token currentToken = docDisplay_.getTokenAt(docDisplay_.getCursorPosition());
```

Comment:
How expensive is this? (if this isn't super cheap we could hide it behind the `c == '('` check to reduce the frequency of invocation

Reply:
Good idea -- while it shouldn't be expensive, it's much better to only do that work if we know we need to.